infrastructure/src/main/java/org/corfudb/infrastructure/datastore/DataStore.java

```diff
@@ -52,48 +66,41 @@
     public static final String EXTENSION = ".ds";
 
     @Getter
-    private final Cache<String, Object> cache;
-    private final String logDirPath;
+    private final Set<String> dsFilePrefixesForCleanup = Sets.newHashSet(
```

Comment:
If this is truly a constant, then fix the naming.

Reply:
`dsFilePrefixesForCleanup` is not static, then it is not a constant, it is just a `final` field